config/smart_answers/rates/child_benefit_rates.yml

```diff
@@ -56,3 +56,8 @@
   end_date: 2023-04-05
   only_child: 21.80
   additional_child: 14.45
+"2023":
+  start_date: 2023-04-06
+  end_date: 2024-04-05
+  only_child: 24.00
+  additional_child: 15.90
```

Comment:
Could we add blank line at the end of this file please @CodeSonia? There are many - mostly old - tools that have issues if an empty blank last line is not present in a file.